server/core/admin.cc

```diff
@@ -939,8 +970,9 @@ bool mxs_admin_init()
                                             !this_unit.using_ssl ? MHD_OPTION_END :
                                             MHD_OPTION_HTTPS_MEM_KEY, this_unit.ssl_key.c_str(),
                                             MHD_OPTION_HTTPS_MEM_CERT, this_unit.ssl_cert.c_str(),
+                                            MHD_OPTION_HTTPS_PRIORITIES, this_unit.ssl_version.c_str(),
                                             this_unit.ssl_ca.empty() ? MHD_OPTION_END :
-                                            MHD_OPTION_HTTPS_MEM_TRUST, this_unit.ssl_cert.c_str(),
+                                            MHD_OPTION_HTTPS_MEM_TRUST, this_unit.ssl_ca.c_str(),
```

Comment:
Please don't include this change in this commit, it's not related to the issue.

It seems to be a regression of [MXS-2956](https://jira.mariadb.org/browse/MXS-2956). If you want, you can submit a separate pull request for it and we'll merge it right away.